src/config.rs

```diff
@@ -2535,6 +2538,7 @@ impl TiKvConfig {
                     + self.raftdb.defaultcf.block_cache_size.0,
             });
         }
+        self.backup.region_max_size = self.coprocessor.region_max_size;
```

Comment:
We should sanitize the value is in a reasonable range. Eg, a large region max size could leads to OOM during backup.

Reply:
Will this happen?